amplify/README.md

```diff
@@ -0,0 +1,8 @@
+# Getting Started with Amplify CLI
+This directory was generated by [Amplify CLI](https://docs.amplify.aws/cli).
+
```

Comment:
Could you expand on what command were generated and what files need to be update? Theres quite a bit checked in here and needed to learn how Amplify  CLI works will be alot of overhead for maintaining this correctly.

Reply:
@elylucas @debrisapron do you all have any input here? I think the only changes that are really needed are the `cli-inputs` because the rename to support federated AWS sign in was needed. The others I am not too sure about